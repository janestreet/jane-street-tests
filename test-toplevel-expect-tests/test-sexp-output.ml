#verbose true;;

let x = 42
[%%expect {|
val x : int = 42
|}];;

[@@@part "blah"];;

6 * 7
[%%expect {|
- : int = 10
|}]
