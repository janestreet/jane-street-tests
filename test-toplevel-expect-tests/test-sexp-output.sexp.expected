((parts
  (((name "")
    (chunks
     (((ocaml_code  "#verbose true;;\
                   \n\
                   \nlet x = 42\
                   \n")
       (toplevel_response  "\
                          \nval x : int = 42\
                          \n")))))
   ((name blah)
    (chunks
     (((ocaml_code "6 * 7\n") (toplevel_response  "\
                                                 \n- : int = 42\
                                                 \n")))))))
 (matched false))
